shared/src/test/scala/mlscript/DiffTests.scala

```diff
@@ -1051,7 +1051,7 @@ class DiffTests
               err.getStackTrace().take(
                 if (mode.fullExceptionStack) Int.MaxValue
                 else if (mode.fixme || err.isInstanceOf[StackOverflowError]) 0
-                else 10
+                else 50
```

Comment:
Why 50? Seems like a lot. Maybe 20?

Reply:
I have added `???` in `toString`, and according to the stack trace, I could find the implicit `toString` call that could not be found by the search. 10 was indeed not enough, so I increased it to 20, and finally to 50, to complete the refactoring. However, I feel that 10 is sufficient normally, so I will change it back to 10.

Reply:
Done.